source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -674,131 +940,20 @@ def test_journey_with_external_uri_from_bragi(self):
-                                },
-                                "zip_codes": [
-                                    "20226"
-                                ]
-                            },
-                            "administrative_regions": [
-                                {
-                                    "id": "admin:fr:02000",
-                                    "insee": "02000",
-                                    "level": 8,
-                                    "label": "Bobtown (02000)",
-                                    "zip_codes": ["02000"],
-                                    "weight": 1,
-                                    "coord": {
-                                        "lat": 48.8396154,
-                                        "lon": 2.3957517
-                                    }
-                                }
-                            ],
-                        }
-                    },
-                    "type": "Feature"
-                }
-            ]
-        }
-
-        bob_street = {
-            "features": [
-                {
-                    "geometry": {
-                        "coordinates": [
-                            0.00188646,
-                            0.00071865
-                        ],
-                        "type": "Point"
-                    },
-                    "properties": {
-                        "geocoding": {
-                            "city": "Bobtown",
-                            "housenumber": "20",
-                            "id": "addr:" + check_utils.r_coord, # the adresse is just above 'R'
-                            "label": "20 Rue Bob (Bobtown)",
-                            "name": "Rue Bob",
-                            "postcode": "02100",
-                            "street": "Rue Bob",
-                            "type": "house",
-                            "citycode": "02000",
-                            "administrative_regions": [
-                                {
-                                    "id": "admin:fr:02000",
-                                    "insee": "02000",
-                                    "level": 8,
-                                    "label": "Bobtown (02000)",
-                                    "zip_codes": ["02000"],
-                                    "weight": 1,
-                                    "coord": {
-                                        "lat": 48.8396154,
-                                        "lon": 2.3957517
-                                    }
-                                }
-                            ],
-                        }
-                    },
-                    "type": "Feature"
-                }
-            ]
-        }
-
         args = {
             u'pt_dataset': 'main_routing_test',
             u'type[]': [u'public_transport:stop_area', u'street', u'house', u'poi', u'city'],
             u'limit': 10,
+            u'depth': 1,
```

Comment:
This should not be necessary as it's default value

Reply:
This parameter in needed for the Mock.